docs/concepts/filesystems.md

```diff
@@ -67,30 +75,111 @@ How can we use `RemoteFileSystem` to store our flow code?
 The following is a use case where we use [MinIO](https://min.io/) as a storage backend:
 
 ```python
-minio_file_packager = FilePackager(
-    filesystem=RemoteFileSystem(
-        basepath="s3://my-bucket",
-        settings={
-            "key": MINIO_ROOT_USER,
-            "secret": MINIO_ROOT_PASSWORD,
-            "client_kwargs": {"endpoint_url": "http://localhost:9000"}
-        }
-    )
+from prefect.filesystems import RemoteFileSystem
+
+minio_block = RemoteFileSystem(
+    basepath="s3://my-bucket",
+    settings={
+        "key": "MINIO_ROOT_USER",
+        "secret": "MINIO_ROOT_PASSWORD",
+        "client_kwargs": {"endpoint_url": "http://localhost:9000"},
+    },
 )
+minio_block.save("minio")
 ```
 
-Now let's look at how we can use `RemoteFileSystem` with AWS S3:
+
+## S3
```

Comment:
for the S3, GCS, and Azure blocks could we note somewhere that if credentials are not explicitly set, they will be inferred from the "standard" environment variables for each file system?

Reply:
both good calls, thanks a lot - will add soon

Reply:
added a section now